sentry-kotlin-multiplatform/sentry_kotlin_multiplatform.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
     spec.name                     = 'sentry_kotlin_multiplatform'
-    spec.version                  = '0.0.1'
+    spec.version                  = '0.0.1-alpha.2'
```

Comment:
I'll have to figure out in another PR to update this automatically as it changes to the project's version during gradle build.

Reply:
For Flutter is always `0.0.1`, so it'd be fine to keep it as it is.